### PR TITLE
Fixes AB#51

### PR DESCRIPTION
**Pull request by peter-murray**
Hi,

This pull request fixes issue(s) <#>.

A summary of changes are:
- adding star rating
 
Please proceed with a review as soon as all the status checks are valid.

### CODE REVIEW

File: src/main/java/com/github/demo/model/Book.java [hunk 1]
```diff
@@ -30,6 +33,14 @@ public Book(String author, String title, String cover) {
         this.cover = cover;
     }
 
+    public Book(String author, String title, String cover, int rating) {
+        this();
+        this.author = author;
+        this.title = title;
+        this.cover = cover;
+        this.rating = rating;
```
Comment (pmanlukas) [inline src/main/java/com/github/demo/model/Book.java:41]: Good addition here

File: src/test/java/com/github/demo/model/BookTest.java [hunk 2]
```diff
@@ -31,6 +31,12 @@ public void testGetCover() {
         Assert.assertEquals("slapstick.jpg", book.getCover());
     }
 
+    @Test
+    public void testGetRating() {
+        book.setRating(5);
```
Comment (pmanlukas) [inline src/test/java/com/github/demo/model/BookTest.java:36]: LGTM 😃

**Review by pmanlukas — APPROVED** [review]
Go ahead, good rating implemention

**Comment by peter-murray** [comment]
@pmanlukas Deploy is good

**Pull request merged**